.travis.yml

```diff
@@ -29,12 +35,10 @@ install:
     - conda env create --name cta --file environment.yml
     - conda activate cta
     - pip install https://github.com/cta-observatory/ctapipe/archive/$CTAPIPE_VERSION.tar.gz
```

Comment:
Why eliminating PROTOZFITS from Travis?

Reply:
because it's not used

Reply:
where?

Reply:
hu? nowhere 😄 

it's legitimate to have `PROTOZFITS` as a requirement of lstchain because most of the user will want to analyse real data (I suppose) but there is 0 unit test using real-data and thus using PROTOZFITS. So I don't see the point of installing it every time for no reason.

Reply:
add PROTOZFITS back to travis